modules/persistent-storage-local-pod.adoc

```diff
@@ -24,10 +24,19 @@ apiVersion: v1
 kind: Pod
 spec:
   ...
+  securityContext:
+    runAsNonRoot: true
+    seccompProfile:
+      type: RuntimeDefault
   containers:
     volumeMounts:
     - name: local-disks <1>
       mountPath: /data <2>
+    # TODO: IS this okay? The container doesn't have a name set, so might not be a complete example
```

Comment:
@s-urbaniak @stlaz This might not be a complete example (the container doesn't have a name). Let me know if you think I should keep this, or pull it out and put some truncation symbols (...) in here instead